languages/go/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,52 @@
+## Packages
+
+In Go an application is organized in packages. A package is a collection of source files located in the same folder. All source files in a folder must have the same package name at the top of the file. The package name is preferred to be the same as the folder it is located in.
+
+```go
+package greeting
+```
+
+## Integers
+
+Go is a statically-typed language, which means that everything has a type at compile-time. Assigning a value to a name is referred to as defining a variable. A variable can be defined either by explicitly specifying its type, or by assigning a value to have the Go compiler infer its type based on the assigned value.
+
+```go
+var explicit int // Explicitly typed
+implicit := 10   // Implicitly typed
+```
+
+The value of a variable can be assigned using the `:=` and updated using the `=`. Once defined, a variable's type can never change.
+
+```go
+count := 1 // Assign initial value
+count = 2  // Update to new value
+
+// Compiler error when assigning different type
+// count = false
+```
+
+## Functions
+
+Functions in Go are considered first class citizens making them very powerful.
```

Comment:
When referring to functions as _first class citizens_, I expected to learn about returning functions and passing functions as arguments. This exercise doesn't teach that (and it shouldn't!) and thus student's don't have to know this to solve the exercise, so maybe this comment could be moved to either the `after.md`  file or to a different exercise?

Reply:
Removed in `introductions.md`. The `after.md` still has it as it also lists all the possibilities functions can be used in Go. @ErikSchierboom: Do you think I should remove it also from after.md (including the list) as there will be another exercise about functions?